### DISCUSSION TITLE
Speed up effect.

### DISCUSSION THREAD

**Discussion by kirkguitar** [hydrogen-music/hydrogen#1782] [General]
Ok, total non-drummer here so be gentle! I'm trying to create the drum pattern as used at the beginning of the Iron Maiden number 'Killers' which begins with some hi hat action. The problem is when it goes from being closed to open, the 'open' effect is too slow. I'm using 'the black pearl' kit and 'sabian hat swish' effect but it seems to go 'pssssit' and I want it to go 'pssit' if that makes any sense at all! If you listen to the song I'm sure you'll get my drift! Is there any way to speed it up? In my defense I've not been using Hydrogen long so I'm still getting to grips with it's power!
Thanks in advance.

**Comment by theGreatWhiteShark** [comment]
Hey @kirkguitar ,

Have you tried setting the note length? First, place a note using left click and then right click and drag the note to the right. This sets a finite length of the note which chokes it in case the custom length is shorter than the sample length. (be sure to set the resolution of the pattern editor high enough as the note length will snap to grid).

This, however, might sound more like "pssss" than "pssit". If you want it more authentically, cutting some milliseconds from the original sample might be the more proper way.

**Reply by kirkguitar** [reply]
Hey thanks for the quick reply! Tried that and it's worked a treat! Didn't even know about the right click option. Many thanks!
